package.json

```diff
@@ -1,39 +1,164 @@
 {
   "name": "react-native-applovin-max",
-  "author": "AppLovin Corporation",
+  "author": "AppLovin Corporation <support@applovin.com> (https://applovin.com)",
   "version": "6.5.0",
   "description": "AppLovin MAX React Native Plugin for Android and iOS",
-  "homepage": "https://github.com/AppLovin/AppLovin-MAX-React-Native",
-  "license": "MIT",
-  "main": "src/index",
+  "main": "lib/commonjs/index",
+  "module": "lib/module/index",
+  "types": "lib/typescript/src/index.d.ts",
   "react-native": "src/index",
   "source": "src/index",
-  "module": "lib/module/index",
   "files": [
     "src",
+    "lib",
     "android",
     "ios",
-    "react-native-applovin-max.podspec",
-    "package.json"
+    "cpp",
+    "*.podspec",
+    "!ios/build",
```

Comment:
What does the `!` mean? Would be curious to get a before-and-after of the built artifact to see if we are adding or removing any files from the package

Reply:
`!` excludes the file(s) from generating a package.   Verified `!` can control the addition/removal of the file.

`cpp` is removed, and added:

```
react-native-applovin-max.podspec
package.json
```